File: practice24/11.cpp

//https://www.nowcoder.com/practice/24ec35c2a8054a7b831a5a3ea660d729?tpId=98&tqId=32875&tPage=3&rp=3&ru=/ta/2019test&qru=/ta/2019test/question-ranking
#include<iostream>
using namespace std;


int main() 
{
	int i, sum, t, sign;
	while (cin >> t)
	{
		sign = 0;
		sum = 0, i = 1;
		while (1)
		{
			sum = sum + i;
			if (sum >= t && (sum - t) % 2 == 0)
			{
				cout << i << endl;
				break;
			}
			i++;
		}
	}
	system("pause");
	return 0;
}

File: practice30/11.java


//https://www.nowcoder.com/practice/45083499b8c5404fb1db44c6ea4f170a?tpId=90&tqId=30845&tPage=4&rp=4&ru=/ta/2018test&qru=/ta/2018test/question-ranking
import java.util.*;
public class Main {
    public static void main(String args[]){
        Scanner scanner=new Scanner(System.in);
        String[] str=scanner.nextLine().split(" ");
        for(int i=0;i<str.length;i++){
            System.out.print(str[i].charAt(0));
        }
        System.out.println();
    }
}